Assets/Scripts/KeyBindings/StoreKeyBindings.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using Newtonsoft.Json;
+
+
+public static class StoreKeyBindings
+{
+    private const string custom_path = "keybindings_custom.json";
+    private const string default_path = "keybindings_default.json";
+
+    static StoreKeyBindings()
+    {
+      
+    }
+        
+    public static void SaveKeyBindings()
+    {
+        Dictionary<String, KeyCode> bindings = KeyBindings.Get();
+
+        var content = JsonConvert.SerializeObject(bindings);
+        File.Delete(custom_path);
+        File.WriteAllText(custom_path, content);
+    }
+
+    public static bool SetDefaultBindings()
+    {
+        if (!File.Exists(custom_path))
```

Comment:
Why does this function return bool, but restore does not return anything?

Reply:
changed as well, though the reason was that it wasn't needed so far